src/commons/stores/cluster-store.js

```diff
@@ -209,11 +209,18 @@
         /****Private Functions****/
 
         function _formatSingleCluster(cluster) {
-            var temp = {};
+            var temp = {},
+                sdsMapping;
+
+            sdsMapping = {
+                "gluster": "Gluster",
+                "rhgs": "RHGS"
+            };
 
             temp.integrationId = cluster.integration_id;
             temp.sdsVersion = cluster.sds_version;
-            temp.sdsName = _getSdsName(cluster.sds_name);
+            temp.sdsName = cluster.sds_name;
+            temp.showSdsName = sdsMapping[cluster.sds_name.toLowerCase()];
```

Comment:
@a2batic why there is need creating separate showSdsName  property. You can do  `temp.sdsName =  sdsMapping[cluster.sds_name.toLowerCase()]`

Reply:
@gnehapk, `sdsName` contains the original value which can be used in future for other purpose, so changing it to display could create problems. Also, if there will be some other `sds_name` later, we would requires to add just a key:value to `sdsMapping`. Also, we are saving a function call here.